test/functional/inheritance_test.rb

```diff
@@ -5,11 +5,11 @@
 
 describe 'example inheritance profile' do
   include FunctionalHelper
-  let(:path) { File.join(examples_path, 'inheritance') }
-  let(:input_file) { File.join(examples_path, 'profile-attribute.yml') } # TODO rename attributes in examples
+  let(:path) { File.join(profile_path, 'inheritance') }
```

Comment:
Ahh, ouch.  I may not have communicated this clearly, but I proposed copying the example profiles (check), then altering [examples_path](https://github.com/inspec/inspec/pull/3889/files#diff-652d43958084a48aa2c78f72b152c98aL58) to point to the new location - which gets you out of having to do all the careful search-and-replace work here (and the many files to follow).  They would still call `examples_path`, but they would point to a different location.

Now we could just accept this (changing `examples_path` -> `profile_path`) but in doing so we lose a lot of information that is valuable for later work.  Down the line, we want to make all the tests that used to rely on examples now rely on dedicated test-fixtures of their own. If we make this change, we can't tell which tests are looks at the old examples location, because many things look at the `profile_path`.

Check in with me when you read this, we can explore this further.

Reply:
after talking with @clintoncwolfe we decided to revert back to using the examples_path variable so we could keep visibility of the tests that use the example profiles; for use when further refactoring is needed